webhook/webhook.go

```diff
@@ -94,6 +94,10 @@ type ControllerOptions struct {
 	// potential races where registration completes and k8s apiserver
 	// invokes the webhook before the HTTP server is started.
 	RegistrationDelay time.Duration
+
+	// ClientAuthType declares the policy the webhook server will follow for
+	// TLS Client Authentication.
+	ClientAuth tls.ClientAuthType
```

Comment:
Can you add a comment to say what this default is when not set from the config? It took me awhile to understand that `0 == tls.NoClientCert` OR I am misunderstanding.

Reply:
yes, the default value of tls.ClientAuthType is tls.NoClientCert. i have added a comment to illustrate it. PTAL